tests/test_connector.py

```diff
@@ -373,15 +376,15 @@ def test__get_object_search_error_return_none(self):
         self.assertEqual(None, self.connector._get_object('network', url))
 
     def test_get_object_with_pagination_with_no_result(self):
-         self.connector._get_object = mock.MagicMock(return_value=None)
-         result = self.connector.get_object('network', paging=True)
-         self.assertEqual(None, result)
+        self.connector._get_object = mock.MagicMock(return_value=None)
+        result = self.connector.get_object('network', paging=True)
+        self.assertEqual(None, result)
 
     def test_get_object_with_pagination_with_result(self):
-         self.connector._get_object = mock.MagicMock(
-                                          return_value={"result": ["data"]})
```

Comment:
isnt this more redable?

Reply:
Yes, but then Flake8 throws an error because the line is too long. I will add a disable flag.